hieradata_aws/class/staging/backend.yaml

```diff
@@ -3,7 +3,5 @@
 icinga::client::check_pings::endpoints:
    whitehall-admin:
      ip: 10.2.3.254
-   content-performance-manager:
-     ip: 10.2.3.254
```

Comment:
Do we need an IP for content-data-api?

Reply:
Same thing applies as the Production configuration https://github.com/alphagov/govuk-puppet/pull/9089#discussion_r281631820